spec/rubocop/cop/layout/heredoc_argument_closing_parenthesis_spec.rb

```diff
@@ -66,6 +66,18 @@
       RUBY
     end
 
+    it 'accepts when existing an argument between heredoc of end and closing parentheses' do
```

Comment:
when there is an argument between a heredoc argument and the closing paretheses - I think this reads better.